src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String PATH_TO_INPUT_FILE = "src/main/resources/";
+
+    public List<String> readFromFile(String fileName) {
+        if (fileName == null || fileName.length() == 0) {
+            throw new RuntimeException("File " + fileName + " is empty");
+        }
+        File file = new File(PATH_TO_INPUT_FILE + fileName);
+        try {
+            List<String> lines = Files.readAllLines(file.toPath());
+            lines.remove(0);
```

Comment:
let's not add any other logic to the reader service, and leave it to the transaction parser, since it is responsible only for reading, and if you will try to read a file for any other purpose you would be surprised why are you missing the first line :)
```suggestion
```

Reply:
ok, I agree with you) thank you)